packages/babel-cli/test/fixtures/babel/dir --out-dir --watch multiple dir/stderr.txt

```diff
@@ -0,0 +1 @@
+EXECUTOR TIMEOUT
```

Comment:
This file should be removed.

(For some reasons this test keeps failing on my local environment and I must accidentally checkout them out)

Reply:
It also keeps failing on my local environment, it's possible _I_ checked it out while rebasing. I spent one hours trying to understand what was wrong with my local env, I'm "happy" that it's not just me 😅